utils/data-migration/bss-ewcs/maps/findPortalValue.js

```diff
@@ -15,7 +15,7 @@ const findFacilityStageValue = (v1Value, v1FieldName, type, logError) => {
   const mappedStage = Object.entries(CONSTANTS.FACILITIES.FACILITIES_STAGE[type]).find((s) => s === v1Value);
   if (v1Value && !mappedStage) {
     logError(`${v1FieldName}: ${v1Value} did not map to v2 FACILTIES_STAGE.${type}`);
-    console.error(`${v1FieldName}: ${v1Value} did not map to v2 FACILTIES_STAGE.${type}`);
+    console.error(`${v1FieldName}: ${v1Value} did not map to v2 FACILTIES_STAGE.${type}`, v1FieldName, v1Value, type);
```

Comment:
Please update, please refer from using string interpolation rather move to string substitution.